blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,228 @@
+:root {
+  --border-color: var(--c-grey-300);
+}
+
+.v2-footer {
+  background: var(--c-background-card);
+}
+
+.v2-footer :is(h3, p, a) {
+  font-family: var(--ff-volvo-novum);
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-size: 16px;
+  letter-spacing: 0.5px;
+}
+
+.v2-footer ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.v2-footer-links-wrapper :is(h3, li, p, a) {
+  margin-bottom: 0;
+  color: var(--c-main-black);
+}
+
+.v2-footer-list h3 {
+  font-weight: bold;
```

Comment:
We should use `font-family: var(--ff-volvo-novum-medium)` instead of `font-weight: bold`. Right now we have faux bold